Classes/FormState/DefaultFormStateInitializer.php

```diff
@@ -31,7 +33,11 @@ public function initializeFormState(FormDefinition $formDefinition, ActionReques
     {
         $serializedFormStateWithHmac = $actionRequest->getInternalArgument('__state');
         if ($serializedFormStateWithHmac !== null) {
-            $serializedFormState = $this->hashService->validateAndStripHmac($serializedFormStateWithHmac);
+            try {
+                $serializedFormState = $this->hashService->validateAndStripHmac($serializedFormStateWithHmac);
+            } catch (InvalidArgumentForHashGenerationException | InvalidHashException) {
+                return new FormState();
+            }
```

Comment:
silently catching and ignoring exceptions is almost never a good idea. IMO we should think about other ways to solve the issue..
What _is_ the issue? That exceptions are logged due to incorrect requests to your site? Can't you detect them otherwise?

Reply:
The problem I try to solve is when somebody tinkers with the hmac in the form, the page will throw a 500er.

Sample: https://www.neos.io/submission-forms/service-provider-submission.html
When manipulating the `--service-provider-form[__state]` in the DOM directly by replacing the value with `foobar` the submission of the form will result in a error 500er page.

This attracts bots hoping to exploit this issue, resulting in thousands of requests.
I know this is not a security issue, at least I don't know how this could get exploit, but still attracts lots of attention.

Reply:
Thanks for the clarification! I agree that a 500 error is a problem in this case, but IMO it should not ignore the error but throw some 4** error

Reply:
Might fix it in a single project, but I'm more looking into a general fix solving it for every project. I think that over 90% of all cases using this package, there is no custom code around this methods.
Just had a look into our sites using this package and all suffer from bots trying to exploit this exception. Fixing it in every single project instead of fixing it at the root feels wrong to me 😅 
I do believe that this exception is only triggered by bad actors, so I only care in them not seeing any exceptions. For normal users this will/should never happen and work just fine.
Maybe discuss this person to person next week? :)

Reply:
I hear you. I'm just concerned that this might hide bugs in someones custom form that creates some invalid state.
OTOH maybe it's just a theoretical issue..

So definitely no -1 from me, let's see what others have to say